.github/workflows/pr-test-e2e.yml

```diff
@@ -183,6 +182,32 @@ jobs:
           curl http://localhost:8080/metrics
           kill %
 
+      - name: Dump redacted Secret YAMLs
+        if: always()
+        run: |
+          echo "--- Dumping redacted Secret YAMLs from context kind-kubeflex ---"
```

Comment:
IMO this header is not necessary, the workflow structure already distinguishes this output from other. Not a blocker.